pkg/cmd/pr/shared/commentable.go

```diff
@@ -82,19 +84,43 @@ func CommentableRun(opts *CommentableOptions) error {
 		return err
 	}
 
+	httpClient, err := opts.HttpClient()
+	if err != nil {
+		return err
+	}
+	apiClient := api.NewClientFromHTTP(httpClient)
+
+	var lastComment *api.Comment
+	if opts.EditLast {
+		username, err := api.CurrentLoginName(apiClient, repo.RepoHost())
+		if err != nil {
+			return err
+		}
+		lastComment, _ = commentable.LastComment(username)
```

Comment:
If the user provided `--edit-last` but does not have a previous comment it feels like this should error out rather than continue on to create a new comment.

Reply:
I think it will be easier to use from the bot by adding a new comment (a.k.a. upsert) if there is no previous comment. Basically, I don't think there are many use cases where users other than bot users use `--edit-last`.
https://github.com/cli/cli/issues/3613#issuecomment-1128685742

Reply:
Ah thanks for the link to the comment, I didn't see the Mislav had suggested this route. Ignore my initial comment.